invenio_app_rdm/config.py

```diff
@@ -652,6 +652,12 @@ def files_rest_permission_factory(obj, action):
 APP_RDM_DEPOSIT_FORM_PUBLISH_MODAL_EXTRA = ""
 """Additional text/html to be displayed in the publish and submit for review modal."""
 
+APP_RDM_DEPOSIT_CAN_HAVE_METADATA_ONLY = True
```

Comment:
What about naming it in a way that explicitly depicts what this variable does i.e 
```suggestion
APP_RDM_DEPOSIT_DISABLE_METADATA_ONLY_REORDS = False
```
or 
```suggestion
APP_RDM_DEPOSIT_REQUIRE_FILES = True
```
@slint @jennur wdyt?

Reply:
Ok let' go with that one then! Can you change the config? I think it should default to `False` so it shows the metadata only section!